build-jenkins-agent-ubuntu.pkr.hcl

```diff
@@ -30,6 +30,32 @@ build {
     ]
   }
 
+  # Retrieve agent.jar
+  provisioner "shell" {
+    only = ["docker.ubuntu"]
+    environment_vars = concat(local.provisioning_env_vars, [
+      "LANG=en_US.UTF-8",
+      "LANGUAGE=en_US:en",
+      "LC_ALL=en_US.UTF-8",
+      "AGENT_WORKDIR=/home/jenkins/agent"
+    ])
+    inline = [
+      "echo Retrieve agent.jar",
+      "curl --create-dirs -fsSLo /usr/share/jenkins/agent.jar https://repo.jenkins-ci.org/public/org/jenkins-ci/main/remoting/${var.remoting_version}/remoting-${var.remoting_version}.jar",
+      "chmod 755 /usr/share/jenkins",
+      "chmod 644 /usr/share/jenkins/agent.jar",
+      "ln -sf /usr/share/jenkins/agent.jar /usr/share/jenkins/slave.jar",
+      "ln -sf /usr/share/jenkins/agent.jar /usr/local/bin/jenkins-agent.jar",
+    ]
+  }
+
+  # Add entrypoint script
+  provisioner "file" {
+    only        = ["docker.ubuntu"]
+    source      = "./provisioning/entrypoint.sh"
```

Comment:
WDYT about also downloading the `entrypoint.sh` from the latest inbound-agent's principal branche on GitHub? 
That would avoid us having to track changes on this script

Reply:
I'll think about it 🙂 
I had to modify it unfortunately, and will have to again.